build_tooling/parallel_test.sh

```diff
@@ -13,12 +13,9 @@ catch=`{ which catchsegv 2>/dev/null || echo ; } | tail -n 1`
 function worker() {
     group=${1:?Must pass the group id argument}
     shift
-    duration_file="$TEST_OUTPUT_DIR/pytest-durations.$group.json"
     new_root=$PARALLEL_TEST_ROOT/$group
     set -o xtrace -o pipefail
 
-    cp "$tooling_dir/pytest-durations.json" "$duration_file"
```

Comment:
Why don't we need this any more? Should remove the checked in `pytest-durations.json` if it is not required

Reply:
After having another thought of it, this should be kept. Will revert.